api/default_networks.go

```diff
@@ -45,7 +45,7 @@ var mainnet = params.Network{
 var goerli = params.Network{
```

Comment:
why don't we have an entry for sepolia here?

Reply:
Good point might be that we added it to the desktop only. We inject config at onboarding and login so we overwrite this. There was a discussion to drop network client injection and rely on status-go, not sure what happened with that.

Reply:
Here it is https://github.com/status-im/status-desktop/issues/11435